stubs/hvac/hvac/constants/approle.pyi

```diff
@@ -0,0 +1,4 @@
+from collections.abc import Iterable
+
+DEFAULT_MOUNT_POINT: str
+ALLOWED_TOKEN_TYPES: Iterable[str]
```

Comment:
Iterable is a strange type for a constant. What is it at runtime?

Reply:
Just a list of strings